recipes/blpapi/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "blpapi" %}
+{% set version = "3.9.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://bloomberg.bintray.com/pip/simple/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 4c3cbf3fb63fd338360d55188f6a59568023be6e16531d4bd307ac59a297fd96
+  - url: https://software.tech.bloomberg/BLPAPI-Stable-Generic/blpapi_cpp_3.8.18.1-linux.tar.gz  # [linux]
+    sha256: c6c425f714b461ec618e6900bda8d9b4fc919892538ccfbf8c9a774e9e234fee  # [linux]
+  - url: https://software.tech.bloomberg/BLPAPI-Stable-Generic/blpapi_cpp_3.8.18.1-windows.zip  # [win]
+    sha256: d62e3f109665f027e2ba3117d015c06a20d5e79e273777ef87915c82311d3fec  # [win]
+  - url: https://software.tech.bloomberg/BLPAPI-Experimental-Generic/blpapi_cpp_3.8.1.1-darwin.tar.gz  # [osx]
+    sha256: 1a74df5f15226aab3fdd7d4f5c765ccfee8cf130d6917962af014a7ac673142f  # [osx]
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - blpapi
+
+about:
+  home: https://www.bloomberg.com/professional/support/api-library/
+  license: Copyright 2012. Bloomberg Finance L.P.
+  license_file: License.txt
```

Comment:
Could you please open a PR to this project to also add the ``License.txt`` in the package?